pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -824,19 +825,19 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 		isInBackOff, msg, err := m.doBackOff(pod, spec.container, podStatus, backOff)
 		if isInBackOff {
 			startContainerResult.Fail(err, msg)
-			klog.V(4).Infof("Backing Off restarting %v %+v in pod %v", typeName, spec.container, format.Pod(pod))
+			klog.V(4).InfoS("Backing Off restarting container in pod", "containerType", typeName, "container", spec.container, "pod", klog.KObj(pod))
```

Comment:
Could we extract name of container from `spec.container` ? I would prefer to use `"containerName", spec.container.Name`.

Reply:
We can extract container name, but isn't it will be a lack of information if we compare to previous log?

Reply:
Ok, looks like original log passed `%+v` with intention to dump whole container object. Passing it here also make sense.